Source/Core/VideoCommon/VideoCommon.h

```diff
@@ -57,6 +63,21 @@ struct TargetRectangle : public MathUtil::Rectangle<int>
 #endif
 };
 
+// This structure should only be used to represent a rectangle in EFB
+// coordinates, where the origin is at the upper left and the frame dimensions
+// are 640 x 528.
+using EFBRectangle = TaggedRectangle<struct EFBRectangleTag>;
```

Comment:
> I'm curious though, what's the reasoning for this?

Mainly because it's analogous with what the [standard library](http://en.cppreference.com/w/cpp/iterator/iterator_tags) does.